assets/espresso/neb.x.in.j2

```diff
@@ -0,0 +1,60 @@
+BEGIN
+BEGIN_PATH_INPUT
+&PATH
+{%- for key, value in path %}
+    {{ typeCast(key, value) }}
+{%- endfor %}
+/
+END_PATH_INPUT
+BEGIN_ENGINE_INPUT
+&CONTROL
+{%- for key, value in control %}
+    {{ typeCast(key, value) }}
+{%- endfor %}
+/
+&SYSTEM
+{%- for key, value in system %}
+    {{ typeCast(key, value) }}
+{%- endfor %}
+/
+&ELECTRONS
+{%- for key, value in electrons %}
+    {{ typeCast(key, value) }}
+{%- endfor %}
+/
+ATOMIC_SPECIES
+{% for species in atomicSpecies -%}
+{{species.X}} {{species.Mass_x}} {{species.PseudoPot_x}}
+{% endfor -%}
+BEGIN_POSITIONS
+FIRST_IMAGE
+ATOMIC_POSITIONS {{atomicPositions.first.card_option}}
+{% for position in atomicPositions.first -%}
+{{position.X}} {{position.x}} {{position.y}} {{position.z}} {% if 'if_pos(1)' in position -%} {{position["if_pos(1)"]}} {% endif -%} {% if 'if_pos(2)' in position -%} {{position["if_pos(2)"]}} {% endif -%} {% if 'if_pos(3)' in position -%} {{position["if_pos(3)"]}} {% endif %}
```

Comment:
@seankwarren why does poisiton has X and x there? what is the difference? I'm just making layout and was confused a bit 
<img width="564" alt="Screenshot 2023-10-11 at 10 23 18 PM" src="https://github.com/Exabyte-io/application-flavors/assets/5090118/47fc7cbb-101a-4d5f-803b-efdbe2f9a8b1">

Reply:
X is the element species based on the docs [here](https://www.quantum-espresso.org/Doc/INPUT_PW.html#idm1493)